platforms/chibios/drivers/spi_master.h

```diff
@@ -72,12 +72,19 @@ typedef int16_t spi_status_t;
 #define SPI_TIMEOUT_IMMEDIATE (0)
 #define SPI_TIMEOUT_INFINITE (0xFFFF)
 
+typedef enum spi_cs_select_mode_t {
+    SPI_CS_SELECT_MODE_NONE,
+    SPI_CS_SELECT_MODE_PAD,
+    SPI_CS_SELECT_MODE_INVERTED,
+} spi_cs_select_mode_t;
```

Comment:
There is some confusion here — there should be only two modes (active low and active high); the “none” case should be handled through `slavePin = NO_PIN`.

Reply:
enum has been removed and replaced with a bool.